docs/installation.md

```diff
@@ -0,0 +1,56 @@
+## Using the boilerplate to create your project
+1. Clone this repository.
+```sh
+git clone https://github.com/ColoredCow/wordpress-init.git
+```
+2. Change remote URL to your project GitHub URL using the following command.
+```sh
+git remote set-url origin https://repository-url
```

Comment:
I think the previous one was fine. Let's also add another command to confirm:
```sh
git remote set-url origin https://github.com/user_name/project_name
git remote -v # will output the project github url if above command ran successfully
```

Reply:
Sounds good.